src/LogicTasks/Syntax/SubTreeSet.hs

```diff
@@ -80,14 +80,8 @@ partialGrade SubTreeInst{..} fs
         english "Your solution contains at least one subformula with more logical operators than the original formula."
         german "Ihre Abgabe beinhaltet mindestens eine Teilformel mit mehr logische Operatoren als die ursprüngliche Formel."
 
-    | amount < minInputTrees =
-      reject $ do
-        english $ "Your solution does not contain enough subformulas. Add " ++ show (minInputTrees - amount) ++ "."
-        german $ "Ihre Abgabe beinhaltet nicht genügend Teilformeln. Fügen Sie " ++ show (minInputTrees - amount) ++ " hinzu."
-
```

Comment:
Dieses Feedback würde ich durchaus in `partialGrade` belassen. Schließlich wird dadurch nichts geleakt. Die Studierenden kennen `minInputTrees` ja bereits aus der Aufgabenstellung.